tests/fp8_kv/README.md

```diff
@@ -0,0 +1,96 @@
+# FP8 KV Cache 
```

Comment:
`tests/fp8_kv/README.md` is most likely an introduction on how to use fp8e4m3 kv cache. It is more useful than `docs/source/quantization/fp8_e4m3_kvcache.rst`. Users can easily access it if we move these to `fp8_e4m3_kvcache.rst`.

Reply:
Sorry, I am not clear about your ask.
You seem to say README.md is easier and useful, but also ask to move README.md down to .rst.

For this ask, may I insist to keep the way it is? Since we came up a long and clear descriptive README.md, moving to .rst can be over kill.

Reply:
Btw, README.md included more general descriptions and tools usages, not limited to kv cache itself.

Reply:
I agree with Zhaoyang that this file does not belong in tests directory. It should be either `docs` or `examples`.

Reply:
`examples` might be better because it directly references the quantize.py code. Make sure to update the links in `docs` as well.